game/scripts/npc/abilities/bristleback_quill_spray.txt

```diff
@@ -58,7 +58,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "max_damage"                                      "550.0 550.0 550.0 550.0 1100.0 2200.0"
+        "max_damage"                                      "150.0 275.0 400.0 550.0 825.0 1100.0"
```

Comment:
Mana potions are 1% as effective as infinity bottles and you know it. Also ferrying Mana potions has a gold cost and a substantial opportunity cost. Opportunity cost in that:
1. you have to micro the courier (no other players can use it)
2. you have to keep an eye on both the hero and the menu and the minimap, splitting your attention 3 ways.
3. you have to click thru the menus which occupies your mind, your mouse, and your eyes
4. you have to keep it in mind that you need an empty slot in your inventory
5. you have to avoid being hit by an enemy hero for the entire duration of the mana potion (30 seconds each?).